unit/java_bytecode/java_types/generic_type_index.cpp

```diff
@@ -0,0 +1,33 @@
+/*******************************************************************\
+
+ Module: Unit tests for java_types
+
+ Author: Diffblue Ltd.
+
+\*******************************************************************/
+
+#include <testing-utils/catch.hpp>
+#include <java_types.h>
+
+SCENARIO("generic_type_index", "[core][java_types]")
+{
+  // Arrange
```

Comment:
More "in style" to use the Catch keywords `GIVEN` `WHEN` and `THEN` instead of arrange/act/assert